Description: disable unused pip
 Remove pip from setup.cfg because it will not be used to build this
 Debian package
Author: Nicholas D Steeves <nsteeves@gmail.com>

---
Forwarded: not-needed
Last-Update: 2020-02-17

--- vorta-0.6.23.orig/setup.cfg
+++ vorta-0.6.23/setup.cfg
@@ -27,7 +27,6 @@ license_file = LICENSE.txt
 [options]
 python_requires = >= 3.6
 setup_requires =
-  pip >= 10
   pytest-runner
   setuptools_git
 install_requires =
